packages/client/src/pages/Game.tsx

```diff
@@ -47,29 +53,50 @@ export const Game = () => {
       if (playerCardNumber === tableCardsSum) {
         toggleMovingCards([activePlayerCard, ...activeCardsOnTable]);
         timer = setTimeout(() => {
-          console.log('Need to emit an event: Cards are going to playerCaptured');
+          console.log('Emit: Cards are going to playerCaptured');
+          gameIO.emit(GameEvent.UpdateState, {
```

Comment:
That's one way of doing it... But I guess all that will happen on server, right? So, client maybe will send a kind of action payload, and server will do that.

Reply:
yeah we can move it on the server, I just wanted to try something